src/styles/getReportActionContextMenuStyles.ts

```diff
@@ -1,12 +1,16 @@
+import {CSSProperties} from 'react';
+import {ViewStyle} from 'react-native';
 import styles from './styles';
 import variables from './variables';
 import themeColors from './themes/default';
 
-const defaultWrapperStyle = {
+type StylesArray = Array<ViewStyle | CSSProperties>;
```

Comment:
It looks like a common type for the entire app. Can we put it somewhere else like general file?

Reply:
@rezkiy37 We are going to standartise this in a follow-up PR after completing styles migration.

Reply:
Cool!